cg/constants/nipt.py

```diff
@@ -1 +1 @@
-Q30_THRESHOLD = 0.88
+Q30_THRESHOLD = 88
```

Comment:
This seems like a bad idea, it is used across many files. We should keep it as is and only adjust it for the specific comparison against the metrics table q30 values

Reply:
Only used for calls to a single function which has been patched